labelbox/client.py

```diff
@@ -117,19 +126,35 @@ def convert_value(value):
                 value = value.strftime("%Y-%m-%dT%H:%M:%SZ")
             return value
 
-        if params is not None:
-            params = {
-                key: convert_value(value) for key, value in params.items()
-            }
-
-        data = json.dumps({'query': query, 'variables': params}).encode('utf-8')
-
+        if query is not None:
+            if params is not None:
+                params = {
+                    key: convert_value(value) for key, value in params.items()
+                }
+            data = json.dumps({
+                'query': query,
+                'variables': params
+            }).encode('utf-8')
+        elif data is None:
+            raise ValueError("Params and data cannot both be none")
```

Comment:
Just reading through this for my own understanding, should this read `Query and data cannot both be none`? Though I suppose if `params` is defined, `query` should be too right? Or am I misreading?

Reply:
This is wrong. Nice catch!